src/test/java/com/puppycrawl/tools/checkstyle/DetailAstImplTest.java

```diff
@@ -641,6 +641,13 @@ public void testToString() {
             .isEqualTo("text[0x0]");
     }
 
+    /**
+     *
+     * @param dir directory to add files from
+     * @return list of files
+     * @noinspection ParameterNameDiffersFromOverriddenParameter
+     * @noinspectionreason false positive
```

Comment:
Add ParameterNameDiffersFromOverriddenParameter to reason.

Reply:
done